src/main/mainWindow.ts

```diff
@@ -34,22 +35,25 @@ import {
     UserAgent,
     VENCORD_FILES_DIR
 } from "./constants";
-import { Settings, VencordSettings } from "./settings";
 import { createSplashWindow } from "./splash";
 import { makeLinksOpenExternally } from "./utils/makeLinksOpenExternally";
 import { applyDeckKeyboardFix, askToApplySteamLayout, isDeckGameMode } from "./utils/steamOS";
 import { downloadVencordFiles, ensureVencordFiles } from "./utils/vencordLoader";
 
 let isQuitting = false;
-let tray: Tray;
 
 applyDeckKeyboardFix();
 
 app.on("before-quit", () => {
     isQuitting = true;
 });
 
-export let mainWin: BrowserWindow;
+// Export a container object of objects that are used by other modules
+// but won't be initialized at import time.
+export const globals = {
```

Comment:
why move these exports into a `global` variable? i don't really like that. i think it was better before

Reply:
`tray` needs to be held in an object of some sort because it can change change or be assigned to null. I figured if I was creating a singleton for `tray` anyway might as well move `mainWindow` in there to clean up the need for require hacks elsewhere

Reply:
you can change let exports
```js
export let tray: BrowserWindow;

// later
tray = newWindow;
```

Reply:
Doesn't this require reimporting the module every time you want to access the up to date object? I guess I'm not actually sure how module import semantics work...

Reply:
no, imports update dynamically. the next time you access the import it will be updated

Reply:
wow, I did not know that's what `export let` did... what the fuck. I'll change it back to the require hack as well then